test/unit/model/Usage/TestUsageServiceTest.php

```
@@ -35,6 +36,42 @@ class TestUsageServiceTest extends TestCase
+
+                return (new \DateTimeImmutable('@' . (int) $timestamp))
+                    ->setTimezone($timeZone)
+                    ->format('d/m/Y H:i:s');
+            }
+
+            public function getFormat($format): string
+            {
+                return 'd/m/Y H:i:s';
+            }
+
+            public function getJavascriptFormat($format): string
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$format'. (undefined)

(UnusedFormalParameter)